package.json

```diff
@@ -20,73 +20,67 @@
     "predev": "yarn generate"
   },
   "dependencies": {
-    "@apollo/client": "^3.8.4",
-    "@atb-as/config-specs": "^3.6.1",
+    "@apollo/client": "^3.8.8",
+    "@atb-as/config-specs": "^3.13.1",
     "@atb-as/theme": "^7.1.1",
     "@github/combobox-nav": "^2.3.0",
     "@leile/lobo-t": "^1.0.5",
     "@mapbox/polyline": "^1.2.1",
-    "@react-aria/focus": "^3.14.2",
+    "@react-aria/focus": "^3.15.0",
     "@react-hook/resize-observer": "^1.2.6",
-    "@types/node": "20.6.2",
-    "@types/react": "18.2.21",
-    "@types/react-dom": "18.2.7",
+    "@types/node": "20.10.4",
+    "@types/react": "18.2.43",
+    "@types/react-dom": "18.2.17",
     "bunyan": "^1.8.15",
-    "cookies-next": "^4.0.0",
+    "cookies-next": "^4.1.0",
     "date-fns": "^2.30.0",
     "detect-nearest-browser-locale": "^19.0.0",
     "detect-nearest-locale": "^1.0.1",
-    "downshift": "^8.2.0",
-    "framer-motion": "^10.16.4",
+    "downshift": "^8.2.3",
+    "framer-motion": "^10.16.16",
     "graphql": "^16.8.1",
     "haversine-distance": "^1.2.1",
-    "humanize-duration": "^3.30.0",
+    "humanize-duration": "^3.31.0",
     "lodash": "^4.17.21",
     "lz-string": "^1.5.0",
-    "mapbox-gl": "^2.15.0",
-    "next": "13.4.19",
+    "mapbox-gl": "^3.0.1",
+    "next": "14.0.4",
     "next-router-mock": "^0.9.10",
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-syntax-highlighter": "^15.5.0",
     "swr": "^2.2.2",
-    "typescript": "5.2.2",
+    "typescript": "5.3.3",
```

Comment:
Is there a reason for why typescript is added to dependencies and not devDependencies? My understanding is that typescript is used in the development phase to maintain better control over the code, but that it is redundant to include it in production, as it is the compiled JavaScript code that is used to run the application.

Reply:
No. To be honest, I don't really know why it is in "dependencies" instead of "devDependencies". But the difference between these actually doesn't impact a whole lot. This only affects wether they should be installed or not, for instance when doing `yarn install --production` you only install `"dependency"` and not `"devDependency"`. But in most cases you only ever do regular installs because you have to build bundles. So in most cases this is only ever used as an indication and sort of semantic categorization in code bases to indicate something on criticality I think. So for instance if something is in `devDependency` you pretty much know that if it compiles or runs locally you are good, and it is easier to remove.

But things included in `dependency` won't automatically be included in production. That is the bundlers task to include things in the compiled code if something refers to it in the code.

But I cleaned up some of the dependencies that for some reason was in `dependency`. (most common reason is forgetting the `-D` flag while installing)

Reply:
Ok, got it! Thanks for the clarification😊